internal/srv/helm.go

```diff
@@ -0,0 +1,68 @@
+package srv
```

Comment:
for down the road... think it would be better to have these clients in their own package, rather than all under the umbrella of `srv`?

Reply:
Oh 💯 . Will definitely be cleaning this up in the near future.